wp-content/mu-plugins/https.php

```diff
@@ -25,25 +26,27 @@ function wp_calculate_image_srcset( $sources ) {
 
 /**
  * Replace http:// with https:// in the embed code (before caching).
- * 
+ *
+ * @since  2.0.0
```

Comment:
@xipasduarte I understand your arguments but I have mixed feelings about this. 

I think we should adopt the WooCrap way... badum tschhh 😝 

Let's see what the "others" have to say. Just remember that we use the [Semantic Versioning](http://semver.org/).